release/models/aft/openconfig-aft-common.yang

```diff
@@ -444,6 +618,120 @@ submodule openconfig-aft-common {
     }
   }
 
+  grouping aft-common-entry-nexthop-mpls-state {
+    description
+      "MPLS encapsulation of a packet.";
+
+    leaf traffic-class {
+      type oc-mplst:mpls-tc;
+      description
+        "The value of the MPLS traffic class (TC) bits, formerly known as the
+         EXP bits.";
+    }
+
+    leaf pop-top-label {
+      type boolean;
+      default false;
+      description
+        "Flag that controls pop action, i.e., the top-most MPLS label
+         should be popped from the packet when switched by the system.
+
+        The top-most MPLS label associated with pop action is equal to
+        the label key used in 'mpls' AFT 'label-entry' list.";
+    }
+
+    leaf-list pushed-mpls-label-stack {
+      type oc-mplst:mpls-label-stack;
+      ordered-by user;
+      description
+        "The MPLS label stack imposed when forwarding packets to the
+         next-hop. See the description of the type
+         oc-mplst:mpls-label-stack for encoding rules.
+
+         Note: a swap operation is reflected by entries in the
+         popped-mpls-label-stack and pushed-mpls-label-stack nodes.";
+    }
+  }
+
+  grouping aft-common-entry-nexthop-encap-udp-state {
+    description
+      "UDP encapsulation applied on top of a packet.";
+
+    leaf src-ip {
+      type oc-inet:ip-address;
+      description
+        "The source IP address for IP/UDP encapsulation.";
+    }
+
+    leaf dst-ip {
+      type oc-inet:ip-address;
+      description
+        "Destination IP address for IP/UDP encapsulation.";
+    }
+
+    leaf ip-dscp {
```

Comment:
Can this just be named `dscp`? I don't think it is ambiguous is it?

Reply:
done